sklearn/tests/test_metaestimators_metadata_routing.py

```diff
@@ -171,6 +176,49 @@ def predict_log_proba(self, X, sample_weight="default", metadata="default"):
         # return np.zeros(shape=(len(X), 2))
 
 
+class ConsumingScorer(_BaseScorer):
+    def __init__(self, registry=None):
+        super().__init__(score_func="test", sign=1, kwargs={})
+        self.registry = registry
+
+    def __repr__(self):
+        return "Consuming_Scorer"
```

Comment:
we don't need a repr, do we?

Reply:
I think this was required because otherwise the meta estimator test fails.

Reply:
I'd be surprised, and doesn't seem to be covered by tests according to codecov.